interfaces/builtin/network_manager.go

```diff
@@ -69,6 +69,10 @@ network packet,
 
 /run/udev/data/* r,
 
+# Allow access to configuration files generated on the fly
+# from netplan
+/run/NetworkManager/** r,
```

Comment:
I suspect this rule would be more future-proof (it will allow you to also list the contents of /run/NetworkManager):
```
/run/NetworkManager/{,**} r,
```

+1 with this change